buildSrc/src/main/kotlin/buildsrc/convention/mockk-publishing.gradle.kts

```diff
@@ -64,13 +64,23 @@ publishing {
             }
         }*/
     }
+    // Configure for Android libraries
+    publications {
+        if (project.extensions.findByName("android") != null) {
+            register<MavenPublication>("release") {
+                afterEvaluate {
+                    from(components["release"])
+                }
+            }
+        }
+    }
     publications.withType<MavenPublication>().configureEach {
         createMockKPom {
             name.set(provider { mavenName })
             description.set(provider { mavenDescription })
         }
 
-        artifact(tasks.provider<Jar>("javadocJar"))
+        artifact(tasks.provider<Task>("javadocJar"))
```

Comment:
The reason for this change is that the following error occurs.

```
> The task 'javadocJar' (org.gradle.jvm.tasks.Jar) is not a subclass of the given type (org.gradle.api.tasks.bundling.Jar).
```

Reply:
It seems to have been caused by importing `org.gradle.jvm.tasks.Jar` in `android-library.gradle.kts`.
Removing import from `android-library.gradle.kts` solved the problem.

Reply:
Nice catch, thanks